packages/desktop/views/dashboard/governance/views/DetailsView.svelte

```diff
@@ -55,27 +56,24 @@
         }
     }
 
-    async function setTotalVotes(): Promise<void> {
-        const participations: Participations = (await getParticipationOverview($selectedAccount?.index))?.participations
-        const selectedProposalOverview = participations[$selectedProposal?.id]
+    async function setCurrentAndTotalVotes(): Promise<void> {
+        const { participations } = (await getParticipationOverview($selectedAccount?.index)) ?? {}
+        const trackedParticipations = Object.values(participations[$selectedProposal?.id] ?? {})
 
-        if (selectedProposalOverview) {
-            const votes = Object.values(selectedProposalOverview).map(
-                ({ amount, startMilestoneIndex, endMilestoneIndex }) => {
-                    const endMilestone = endMilestoneIndex <= 0 ? $networkStatus?.currentMilestone : endMilestoneIndex
-                    return parseInt(amount, 10) * (endMilestone - startMilestoneIndex)
-                }
-            )
-            totalVotes = votes?.reduce((accumulator, votes) => accumulator + votes, 0) ?? 0
-        } else {
-            totalVotes = 0
-        }
+        const votes = trackedParticipations.map(({ amount, startMilestoneIndex, endMilestoneIndex }) => {
+            const endMilestone = endMilestoneIndex <= 0 ? $networkStatus?.currentMilestone : endMilestoneIndex
+            return parseInt(amount) * (endMilestone - startMilestoneIndex)
+        })
```

Comment:
Might be worth putting this into its own `utils` function in the `governance` module.